src/styles/_catalog.scss

```diff
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-  }
+    transition: transform 0.3s, color 0.3s;
```

Comment:
u use transform duration 2 times, it will be better to make a vabiable.

Reply:
yeah, maybe in this task it will be good, but I don't think like that about some huge projects, because someone might want to set them different values sometime. I guess it store in one place, so you can easily find this information and change it, why do we need to create another one "in the one place" as variables